packages/live-preview-sdk/src/inspectorMode/utils.ts

```diff
@@ -58,89 +58,92 @@ export function getInspectorModeAttributes(
 /**
  * Query the document for all tagged elements
  */
-export function getAllTaggedElements(root = window.document, ignoreManual?: boolean): Element[] {
+export function getAllTaggedElements(_root = window.document, _ignoreManual?: boolean): Element[] {
```

Comment:
This would also break the manual tagging

Reply:
Then let's merge it as chore to not release a version for it?

Reply:
the next PR should be then released as canary again

Reply:
yes both PRs are going to be canary releases